File: socailproject/modal/FollowModal.java
package com.socaiproject.socailproject.modal;

import lombok.Data;

@Data
public class FollowModal {

    Long id;
    int count;
    String uid;
    String userName;
    String followUser;

    public FollowModal(Long id, int count, String uid, String userName ,String followUser) {
        this.id = id;
        this.count = count;
        this.uid = uid;
        this.userName = userName;
        this.followUser=followUser;
    }

    public FollowModal() {

    }

    public String getFollowUser() {
        return followUser;
    }

    public void setFollowUser(String followUser) {
        this.followUser = followUser;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }
}

File: app/src/main/java/com/example/rahul/modal/FollowModal.kt
package com.example.rahul.modal

data class FollowModal(
    var id: Long? =0,
    var count: Int =0,
    var uid: String ="",
    var userName: String ="",
    var followUser: String ="")

{
    constructor(count: Int,uid: String,userName: String,followUser: String):this(null,count,uid,userName,followUser)
    constructor(uid: String,userName: String,followUser: String):this(null,0,uid,userName,followUser)

}